spec.yaml

```diff
@@ -529,6 +529,7 @@ definitions:
                 Warning: client implementers should be aware that v1.1 servers
                 MAY support both `webdav` and `multi`, but v1.0 servers MAY
                 only support `webdav`.
+                This field may be removed in a future major version of the spec.
```

Comment:
```suggestion
                Default: `multi`. This field may be removed in a future major version of the spec.
```

Reply:
It's clear to us but maybe not to the reader, that if this field is omitted, then `multi` is assumed.